### PR TITLE
Add V0.47.0.1 branch

### PR DESCRIPTION
**Pull request by ReMoose**
This is the version that is currently used in ULLINK.
In the first two commits I have made it possible for the nant build to be done using .Net Framework 2.0 (as this old version of IKVM isn't compatible with .Net 4.0), and added the suggestion that the build should be done with Java 6.
Afterwards, I have added the commits that were done by ULLINK on top of 0.46.0.1.

### CODE REVIEW

**Comment by ReMoose** [comment]
A new branch needs to be created on the Ullink repo for this version. Will be discussed.

File: HOWTO [hunk 1]
```diff
@@ -45,3 +45,5 @@ The version numbers of the build is defined in ikvm/CommonAssemblyInfo.cs.
 
 If you want to do a strong named build, make sure you have a key container installed named
 "ikvm-key" and run the build with "nant signed".
+
+If the build fails, try to build it using Java 6.
```
Comment (tephe) [inline HOWTO]: We can move this in the "Windows:" section or before it as: Make sure you run the build using Java 6

Comment (ReMoose) [reply HOWTO]: Done

File: runtime/openjdk.cs [hunk 2]
```diff
@@ -7943,9 +8237,28 @@ static class NTSystem
 	{
 		public static void getCurrent(object thisObj, bool debug)
 		{
-			throw new NotImplementedException();
-		}
-	}
+            System.Security.Principal.WindowsIdentity id = System.Security.Principal.WindowsIdentity.GetCurrent();
```
Comment (timotei) [inline runtime/openjdk.cs:8377]: Invalid formatting

Comment (ReMoose) [reply runtime/openjdk.cs:8377]: Done

File: IKVM.0.47.0.1.nuspec [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        <version>0.47.0.1</version>
+        <authors>Ullink</authors>
+        <owners>Ullink</owners>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <description>IKVM 0.47 compiled with Ullink Key</description>
+        <summary />
+        <language />
+    </metadata>
+    <files>
+        <file src="bin\IKVM.AWT.WinForms.dll" target="lib\net35\IKVM.AWT.WinForms.dll" />
```
Comment (timotei) [inline IKVM.0.47.0.1.nuspec:14]: Should be net20, no?

**Review by timotei — APPROVED** [review]

File: IKVM.0.47.0.1.nuspec (hunk 3 above)
Comment (timotei) [reply IKVM.0.47.0.1.nuspec:14]: It seems we already bundled it like that. For the sake of starting from exactly where we are, we can keep it like this :)

Comment (ReMoose) [reply IKVM.0.47.0.1.nuspec:14]: hmm, it seems so, but this is the way it is for our internal repo and maybe it should stay this way 😄

**Pull request merged**